lib/game_teams_stats.rb

```diff
@@ -7,6 +7,62 @@ class GameTeamStats
 
   def initialize(file_path, object)
     @game_teams = csv_data(file_path, object)
+    @team_stats = TeamStats.new("./data/teams.csv", Team)
   end
 
+  def lowest_scoring_visitor
+    scoring('away','low')
+  end
+
+  def lowest_scoring_home_team
+    scoring('home','low')
+  end
+
+  def highest_scoring_visitor
+    scoring('away','win')
+  end
+
+  def highest_scoring_home_team
+    scoring('home','win')
+  end
```

Comment:
This astounds me. Great job with using a single helper method for simplify each of these stat methods.